cmd/tetra/tracingpolicy/tracingpolicy.go

```diff
@@ -77,7 +77,7 @@ func New() *cobra.Command {
 	tpListFlags.StringVarP(&tpListOutputFlag, common.KeyOutput, "o", "text", "Output format. text or json")
 
 	tpGenerateCmd := &cobra.Command{
-		Use:   "generate",
+		Use:   "generate <all-syscalls|empty>",
```

Comment:
we could use `cobra.Command.ValidArgs` that would provide more feedback to the user but it's a nit.

Reply:
ok, I'm adding more sub commands in following PR, I'll see if I can use that, thanks